package.json

```diff
@@ -21,9 +21,9 @@
     "build": "npm run bundle && npm run format && npm run minify && npm run test",
```

Comment:
Can we please just not have npm run test as part of the build any more (for any repo)? Bombs for me (and others) because of IE.

Reply:
I don't have a problem with dropping `test-page`; Webcharts hasn't changed though, still just running unit tests.  Personally I'd like to also drop `test` from `build` and because Travis CI runs the tests anyway and it's time-consuming to run them manually.